cvat-ui/src/components/annotation-page/review/conflicts-aggregator.tsx

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2023 CVAT.ai Corporation
```

Comment:
Almost full dublicate of ``issues-aggregator.tsx``. 
I believe better to add the loop ``for (const conflict of mapping) {`` to there

Reply:
Moved all logic to issues-aggregator